civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1095,6 +1135,13 @@ function civihr_employee_portal_views_query_alter(&$view, &$query) {
     case MyDetailsView::getName():
       $viewClass = new MyDetailsView();
       break;
+    case 'mydetails_myrole':
```

Comment:
seems that we have a pattern of having classes representing individual views, we should then follow it also for this "mydetails_myrole" view as well, as it would make the code cleaner

Reply:
Honestly it seems to me an overkill to have a class for a simple modification, in that case trying to encapsulate code is adding more complexity, not simplifying code.

Reply:
It is done with a class now.